dubbo-remoting/dubbo-remoting-netty4/src/main/java/org/apache/dubbo/remoting/transport/netty4/NettyConnectionClient.java

```diff
@@ -131,7 +131,8 @@ protected void initChannel(SocketChannel ch) {
 
                 NettyConfigOperator operator = new NettyConfigOperator(nettyChannel, getChannelHandler());
                 protocol.configClientPipeline(getUrl(), operator, nettySslContextOperator);
-                ch.closeFuture().addListener(channelFuture -> doClose());
+                // set null but do not close this client, it will be reconnect in the future
+                ch.closeFuture().addListener(channelFuture -> channel.set(null));
```

Comment:
Seems this change with problem, not sure if only set channel status to null will cause problem or not? Maybe need dubbo or netty expert to help to answer.

Suggest to evaluate and test this change again.

Reply:
We downgrade to 3.2.10, till now, all works fine.

Reply:
Confirmed that the 3.2.10 resolved our problems, no errors for connection.